src/components/projects/projects-table/projects-table.tsx

```diff
@@ -33,7 +95,90 @@ const ProjectsTable: React.FC = () => {
     }) as DisplayProject[];
   };
 
-  return <PrjsTable allProjects={transformToDisplayProjects(data!)} />;
+  /**
+   * Updates state with data from input parameters.
+   * @param status The status of the project.
+   * @param year The year the project was created.
+   * @param projectLead The project lead of the project.
+   * @param projectManager The project manager of the project.
+   * @param carNumber The carNumber of the project.
+   */
+  const sendDataToParent = (
+    status: string,
+    year: string,
+    projectLead: string,
+    projectManager: string,
+    carNumber: string
+  ) => {
+    setStatus(status);
+    setYear(year);
+    setProjectLead(projectLead);
+    setProjectManager(projectManager);
+    setCarNumber(carNumber);
+  };
+
+  /**
+   * Returns an array of user names who are listed as a project's lead.
+   */
+  const getLeads = (): string[] => {
+    const projects = data!;
+    let leads: string[] = [];
+    for (let project of projects) {
+      let name = fullNamePipe(project.projectLead);
```

Comment:
very soon the `project.projectLead` field will be an array

Reply:
I think we will have to wait until that other branch that deals with that is merged to make the changes in this line.